beacon-chain/state/stategen/service.go

```diff
@@ -55,14 +56,14 @@ type State struct {
 type saveHotStateDbConfig struct {
 	enabled         bool
 	lock            sync.Mutex
-	duration        uint64
+	duration        types.Slot
```

Comment:
`duration` should be a number.

Reply:
No, it is not a number. Duration is in slots, so should be `types.Slot` (which actually helps us by looking at type know what units the `duration` is measured in).